openkfw/TruBudget
# 1841 Add tooltip to default assignee selection

### Checklist




### Description


### How to test

1. Open any project
2. Open subproject create dialog
3. Check if tooltip next to "default assignee" select is working properly
4. ...




Closes #1841
